include/dem/particle_wall_contact_force.h

```diff
@@ -12,9 +12,6 @@
  * the top level of the Lethe distribution.
  *
  * ---------------------------------------------------------------------
-
- *
- * Author: Shahab Golshan, Polytechnique Montreal, 2019
```

Comment:
Question: should we do this for all files? @blaisb . Sometimes I hesitate doing this because I don't know